client/utilities/hooks/useConsent.ts

```diff
@@ -7,7 +8,8 @@ export const useConsent = () => {
 			cmp.init({
 				// Default to GB so it works when no geolocation
 				// cookie is present (eg. local development)
-				country: getGeoLocation() ?? 'GB',
+				country:
+					(getGeoLocation() as CountryCode) ?? ('GB' as CountryCode),
```

Comment:
That might be a much larger change as I suspect that getGeoLocation is used in quite a few places which then all would have to change

Reply:
I was going to suggest the same thing as @cemms1, but if that's problematic would it make sense to cast once here:

```suggestion
				country:
					(getGeoLocation() ?? 'GB') as CountryCode,
```

I think that better reflects what's happening here. Casting the return value of `getGeoLocation()` directly feels slightly risky as we know that it can return `null` so casting after falling back on a default seems a little better. What do you think?